Basic-Recruit-Training/README.adoc

```diff
@@ -1,8 +1,92 @@
-Basic-Recruit-Training
------------------------
+= Pointers
```

Comment:
I think it would make sense to split this README into two; one in the `Person` exercise and one for the `Pointer` exercise.

This readme would then be an overview of the entire Basic-Recruit-Training` - one that talks about the content of each folder.